src/runtime-rs/crates/resource/src/network/network_model/route_model.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::process::Command;
+
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+
+use super::{NetworkModel, NetworkModelType};
+use crate::network::NetworkPair;
+
+#[derive(Debug)]
+pub(crate) struct RouteModel {}
+
+impl RouteModel {
+    pub fn new() -> Result<Self> {
+        Ok(Self {})
+    }
+}
+
+#[async_trait]
+impl NetworkModel for RouteModel {
+    fn model_type(&self) -> NetworkModelType {
+        NetworkModelType::Route
+    }
+
+    async fn add(&self, pair: &NetworkPair) -> Result<()> {
+        let tap_name = &pair.tap.tap_iface.name;
+        let virt_name = &pair.virt_iface.name;
+
+        //setup rule
+        let output = Command::new("ip")
+            .args(&[
+                "rule", "add", "pref", "10", "from", "all", "lookup", "local",
+            ])
+            .output()
+            .context("ip rule add pref 10 from all lookup local")?;
+        if !output.status.success() {
+            return Err(anyhow!("{}", String::from_utf8(output.stderr)?));
+        }
+
+        let output = Command::new("ip")
```

Comment:
Same with above

Reply:
and belows

Reply:
Thanks for your review, change to `tokio::process::Command`